src/pages/styledPages/StyledCardContainer.styled.tsx

```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components";
+
+export const StyledCardContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  background: #cfe7fa;
+  );
```

Comment:
Milos please do self code review in the future, these kinds of mistake shoudn't happen this often

Reply:
I will do that in the future, this kind of mistake  will be reduced to a minimum